### PR TITLE
docs(PPDSC-2006): shrink all title and desc containers for docs

### PR DESCRIPTION
**Pull request by evgenitsn**
PPDSC-2006

**What**

1. Background - why this is needed: Design updates
2. What did you do - Updated the `content-base.tsx` component to handle 12 columns grid and have the text blocks to be 10 out of these 12 cols.
3. What does the reviewers should expect - updated text blocks layout for the foundation pages.

**I have done:**
- [ ] Written unit tests against changes
- [ ] Written functional tests against the component and/or NewsKit site
- [ ] Updated relevant documentation

**I have tested manually:**
- [x] The feature's functionality is working as expected on Chrome, Firefox, Safari and Edge
- [ ] The screen reader reads and flows through the elements as expected.
- [x] There are no new errors in the browser console coming from this PR.
- [x] When visual test is not added, it renders correctly on different browsers and mobile viewports (Safari, Firefox, small mobile viewport, tablet)
- [ ] The Playground feature is working as expected

| **Before:**  | **After:** |
| ----------- | ----------- |
| image      | image|

**Who should review this PR:**

**How to test:**

All content foundation pages should be now updated and ready for review **except** the `/getting-started/design/design-quickstart` because it uses an `OrderedList` component which contains a mix of images and text values which we cannot split to 12 vs 10 columns at the moment. This has to be reworked.

Here is a **non exhaustive** list of pages that have been updated:
- `/getting-started/code/instrumentation/`
- `/getting-started/code/engineering-quickstart/`
- `/theme/borders/`
- `/theme/fonts/`
- `/theme/spacing/`
- `/theme/colours/`
- `/about/contact-us/`
- `/about/contribute/`
- `/about/roadmap/`

### CODE REVIEW

**Comment by newskit-engineering** [comment]
You can preview these changes on: 
  Documentation Site: http://ncu-newskit-docs.s3-website-eu-west-1.amazonaws.com/ppdsc-2006-shrink-all-title-and-desc-containers-for-docs-foundation-pages 
  Storybook: http://ncu-newskit-docs.s3-website-eu-west-1.amazonaws.com/ppdsc-2006-shrink-all-title-and-desc-containers-for-docs-foundation-pages/storybook

**Comment by Vanals** [comment]
Just a remainder for changing the base to `main`!  ;)

File: site/pages/getting-started/code/instrumentation.tsx [hunk 1]
```diff
@@ -46,7 +47,12 @@ const customEventRows = [
 ];
 
 const contentOverrides = {
-  typographyPreset: 'editorialParagraph030',
+  typographyPreset: {
+    xs: 'editorialParagraph020',
+    md: 'editorialParagraph030',
+    lg: 'editorialParagraph030',
+    xl: 'editorialParagraph030',
+  },
```
Comment (mutebg) [inline site/pages/getting-started/code/instrumentation.tsx:55]: you can do like that to keep it shorter:
```suggestion
  typographyPreset: {
    xs: 'editorialParagraph020',
    md: 'editorialParagraph030',
  },
```

File: site/components/tabs-with-table/tabs-with-table.tsx [hunk 2]
```diff
@@ -1,26 +1,30 @@
...
-              <Table columns={columnHeader} rows={rows} />
-            </>
+            <GridLayout columns={`repeat(${ContentColSpan.FULL}, 1fr)`}>
+              <GridLayoutItem column={`auto / span ${ContentColSpan.TEXT}`}>
+                <ContentText>{description}</ContentText>
+              </GridLayoutItem>
+              <GridLayoutItem column={`auto / span ${ContentColSpan.FULL}`}>
+                <Table columns={columnHeader} rows={rows} />
+              </GridLayoutItem>
+            </GridLayout>
```
Comment (mutebg) [inline site/components/tabs-with-table/tabs-with-table.tsx:27]: I just wonder if that content should be something like `ContentTertiary`?

**Review by mutebg — COMMENTED** [review]
2 small comments, the rest looks very good, well done with this ticket

File: site/components/tabs-with-table/tabs-with-table.tsx (hunk 2 above)
Comment (evgenitsn) [reply site/components/tabs-with-table/tabs-with-table.tsx]: Yup, I've updated it 🚀

**Review by mutebg — APPROVED** [review]

**Review by Xin00163 — DISMISSED** [review]
As it's docs change only, the branch name starts with `docs/`. Other than that, looks good to me.

**Comment by evgenitsn** [comment]
@Xin00163 @mutebg After this change d1c7a79e9fc3c426d234a42fd463905f71dd91f3 I notices that there an additional space has been added below the `<TabsWithTable/>` component. That's because a `<ContentTertiary/>` component is nested into a `<ContentSecondary/>` and this leads to double bottom spacing. 
I've added an optional prop for the `<ContentBase/>` which would handle cases like this one and will remove the double spacing. 

Ready for another review.

**Pull request merged**